dimscord/helpers/channel.nim

```diff
@@ -40,11 +39,11 @@ template create*(g: Guild, what: typedesc[GuildChannel];
         user_limit, permission_overwrites, reason
     )
 
-template delete*(ch: GuildChannel, reason = ""): Future[void] =
+template deleteChannel*(ch: SomeChannel, reason = ""): Future[void] =
```

Comment:
We already know its a channel from the signature, wouldn't `delete` make more sense instead of the unneeded "Channel" suffix?